src/event_model/tests/test_run_router.py

```diff
@@ -340,7 +340,7 @@ def exception_callback_factory(start_doc_name, start_doc):
     rr("start", start_doc)
 
     descriptor_doc = {"run_start": "abcdef", "uid": "ghijkl"}
-    with pytest.raises(Exception):
+    with pytest.raises(UserWarning):
```

Comment:
Why the change? If this changed to a warning shouldn't the test use `pytest.warns`?

Reply:
The change was just because linting wasn't a fan of`pytest.raises` with a base `Exception`, but `rt("descriptor", descriptor_doc)` both raises `Exception` and `warns` so the `pytest.raises(UserWarning)` erroneously caught it.